src/services/api/Authenticator.js

```diff
@@ -6,18 +6,43 @@ import React from 'react';
  */
 
 export default function Authenticator() {
-  let manager = new UserManager({
-    automaticSilentRenew: true,
-    response_type: 'code',
-    filterProtocolClaims: true,
-    scope: 'openid ',
-    loadUserInfo: true,
-    client_secret: window.OIDC_CLIENT_SECRET,
-    authority: window.OIDC_PROVIDER_URL,
-    client_id: window.OIDC_CLIENT_ID,
-    redirect_uri: `${window.OIDC_REDIRECT_URI}/callback`,
-    post_logout_redirect_uri: `${window.OIDC_REDIRECT_URI}/logout`
-  });
+
+  let manager = null;
+
+  console.log(window.OIDC_PROVIDER_URL);
+
+  if (window.OIDC_CLIENT_ID === 'undefined') {
+    window.OIDC_CLIENT_ID = OIDC_CLIENT_ID;
+  }
+
+  if (window.OIDC_PROVIDER_URL === 'undefined') {
+    window.OIDC_PROVIDER_URL = OIDC_PROVIDER_URL;
+  }
+
+  if (window.OIDC_CLIENT_SECRET === 'undefined') {
+    window.OIDC_CLIENT_SECRET = OIDC_CLIENT_SECRET;
+  }
+
+  if (window.OIDC_REDIRECT_URI === 'undefined') {
+    window.OIDC_REDIRECT_URI = OIDC_REDIRECT_URI;
+  }
+
+  console.log(window.OIDC_PROVIDER_URL, OIDC_PROVIDER_URL);
```

Comment:
Is this necessary?

Reply:
No, it just for testing. I will remove it later.

Reply:
Sorry I haven't seen that the PR was still not ready.